bundles/org.openhab.core.ephemeris/NOTICE

```diff
@@ -0,0 +1,19 @@
+This content is produced and maintained by the Eclipse SmartHome project.
+
+* Project home: https://eclipse.org/smarthome/
```

Comment:
Please update this to an openHAB NOTICE file.

Reply:
I have updated it to the openHAB notice, but as we have Jollyday only as a dependency, I don't think this needs to be added here - all code in this bundle is EPL.